src/assignment/assignment.service.ts

```diff
@@ -875,14 +871,15 @@ export class AssignmentService {
 						)
 					);
 				}
-				return null;
+				return existingAssignmentResponses[0];
 			}
 
 			// Student has never viewed this assignment before, we should create a response object for him
 			const assignmentResponse: Partial<Avo.Assignment.Response_v2> = {
 				owner_profile_id: getProfileId(user),
 				assignment_id: assignment.id,
-				collection_title: null,
+				collection_title:
+					assignment.assignment_type === AssignmentType.BOUW ? 'Nieuwe collectie' : null,
```

Comment:
Bij het aanmaken van opdrachten heb ik de default value mee in translations gestoken als future-proofing en om controle van die value bij meemoo te leggen.